sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -628,11 +628,19 @@ class SdkSynchronizer internal constructor(
 
     override suspend fun cancelSpend(pendingId: Long) = txManager.cancel(pendingId)
 
+    // TODO(str4d): Rename this to getCurrentAddress (and remove/add in changelog).
+    /**
+     * Returns the current Unified Address for this account.
+     */
```

Comment:
I added this comment when I thought this was the public API that led to `RustWelding.getShieldedAddress`, but it does not. I am still figuring out how it is populated, and what the access patterns are.

Reply:
We shouldn't change addresses too much just yet; there are changes happening in the Roomoval PR to make these more type safe.